tests/utils/GraphicalUnitTester.py

```diff
@@ -76,7 +76,7 @@ def _load_data(self):
             The pre-rendered frame.
         """
         frame_data_path = os.path.join(
-            os.path.join(self.path_control_data, "{}.npz".format(str(self.scene)))
+            os.path.join(self.path_control_data, f"{str(self.scene)}.npz")
```

Comment:
This could also be changed to pathlib, but this can also happen in a future pr :)

Reply:
Good point. I would prefer to not mix those two things.